tests/integration/conftest.py

```diff
@@ -160,13 +161,23 @@ async def app_fixture(
             config=app_config,
             series="focal",
         )
+
+    await model.wait_for_idle()
```

Comment:
Perhaps specifying the status to wait for can be useful to debug when this fails

Reply:
status added